[2024-04-01T09:57:29.755+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag___dbt.dbt_data_quality_checks scheduled__2024-03-30T00:00:00+00:00 [queued]>
[2024-04-01T09:57:29.780+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag___dbt.dbt_data_quality_checks scheduled__2024-03-30T00:00:00+00:00 [queued]>
[2024-04-01T09:57:29.783+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2024-04-01T09:57:29.833+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): dbt_data_quality_checks> on 2024-03-30 00:00:00+00:00
[2024-04-01T09:57:29.844+0000] {standard_task_runner.py:60} INFO - Started process 712 to run task
[2024-04-01T09:57:29.853+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag___dbt', 'dbt_data_quality_checks', 'scheduled__2024-03-30T00:00:00+00:00', '--job-id', '130', '--raw', '--subdir', 'DAGS_FOLDER/dbt_dag.py', '--cfg-path', '/tmp/tmpbw5v_swe']
[2024-04-01T09:57:29.859+0000] {standard_task_runner.py:88} INFO - Job 130: Subtask dbt_data_quality_checks
[2024-04-01T09:57:30.087+0000] {task_command.py:423} INFO - Running <TaskInstance: dag___dbt.dbt_data_quality_checks scheduled__2024-03-30T00:00:00+00:00 [running]> on host 62dc842f1ceb
[2024-04-01T09:57:30.445+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag___dbt' AIRFLOW_CTX_TASK_ID='dbt_data_quality_checks' AIRFLOW_CTX_EXECUTION_DATE='2024-03-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-30T00:00:00+00:00'
[2024-04-01T09:57:30.451+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-01T09:57:30.456+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/***/.local/bin/dbt test --profiles-dir /home/***/.dbt --project-dir /opt/***/proj']
[2024-04-01T09:57:30.507+0000] {subprocess.py:86} INFO - Output:
[2024-04-01T09:57:53.928+0000] {subprocess.py:93} INFO - [0m09:57:53  Running with dbt=1.7.10
[2024-04-01T09:57:55.665+0000] {subprocess.py:93} INFO - [0m09:57:55  Registered adapter: postgres=1.7.10
[2024-04-01T09:58:16.262+0000] {local_task_job_runner.py:302} WARNING - State of this instance has been externally set to removed. Terminating instance.
[2024-04-01T09:58:16.291+0000] {process_utils.py:131} INFO - Sending 15 to group 712. PIDs of all processes in the group: [713, 712]
[2024-04-01T09:58:16.298+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 712
[2024-04-01T09:58:16.308+0000] {taskinstance.py:2450} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-04-01T09:58:16.320+0000] {subprocess.py:104} INFO - Sending SIGTERM signal to process group
[2024-04-01T09:58:16.566+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=712, status='terminated', exitcode=0, started='09:57:29') (712) terminated with exit code 0
[2024-04-01T09:58:16.578+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=713, status='terminated', started='09:57:29') (713) terminated with exit code None
